includes/class-wc-admin-reports-sync.php

```diff
@@ -132,6 +132,11 @@ public static function schedule_single_order_process( $order_id ) {
 			return;
 		}
 
+		if ( apply_filters( 'woocommerce_disable_scheduling', false ) ) {
```

Comment:
Optional nitpick: should this filter name include "order" since it only prevents order updates from being queued?

Reply:
Hmm, good point.  My thought process was that we may want to use this same hook to easily allow disabling across wc-admin and maybe eventually core so I opted to keep it generic.

But since we don't have this anywhere else right now, it may be a good idea to make it more descriptive.  I'll update to `woocommerce_disable_order_scheduling` and we can modify in the future if we want to disable scheduling elsewhere.